README.md

```diff
@@ -51,7 +51,7 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
-### Please [star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀
+### Please [star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀.
```

Comment:
Remove `.`

Reply:
Should I make a new pull request in order to resolve these changes ?

Reply:
done